hydra-node/src/Hydra/Chain/Direct/State.hs

```diff
@@ -472,7 +481,13 @@ observeInit ::
   Tx ->
   Maybe (OnChainTx Tx, InitialState)
 observeInit ctx tx = do
-  observation <- observeInitTx networkId (allVerificationKeys ctx) ownParty tx
+  observation <-
+    observeInitTx
+      networkId
+      (allVerificationKeys ctx)
+      (Hydra.Chain.Direct.State.contestationPeriod ctx)
```

Comment:
Could: Access by record field in line 505 to make this less awkward.

Reply:
it clashes with `InitObservation` `contestationPeriod`